sematic/api/endpoints/meta.py

```diff
@@ -43,12 +45,29 @@ def env_endpoint(user: Optional[User]) -> flask.Response:
     env = {}
     for settings in (
         ServerSettingsVar.GOOGLE_OAUTH_CLIENT_ID,
-        ServerSettingsVar.KUBERNETES_NAMESPACE,
         ServerSettingsVar.GRAFANA_PANEL_URL,
+        ServerSettingsVar.SEMATIC_RESOURCE_UPDATE_POLL_INTERVAL_SECONDS,
     ):
         try:
             env[settings.value] = get_server_setting(settings)
         except MissingSettingsError:
             continue
 
     return flask.jsonify(dict(env=env))
+
+
+@sematic_api.route("/api/v1/meta/env/resource_poll_interval_seconds", methods=["GET"])
+@authenticate
+def resource_poll_interval_seconds_endpoint(user: Optional[User]) -> flask.Response:
+    """Return a dict containing the interval between polls for resource updates."""
+    interval = _DEFAULT_INTERVAL_BETWEEN_RESOURCE_UPDATES_SECONDS
+    try:
+        interval = int(
+            get_server_setting(
+                ServerSettingsVar.SEMATIC_RESOURCE_UPDATE_POLL_INTERVAL_SECONDS
+            )
+        )
+    except MissingSettingsError:
+        pass
+
+    return flask.jsonify(dict(value=interval))
```

Comment:
Can we just hard-code this? This adds too much complexity. Adding an endpoint to get a single settings seems overkill. Plus it is returned in the previous endpoint is it not?

But in general, let's just hard-code this until we need to make it configurable. I don't see a need for it at this time.